java/src/jmri/jmrit/roster/swing/speedprofile/SpeedProfilePanel.java

```diff
@@ -59,6 +61,7 @@
     JButton testButton = new JButton(Bundle.getMessage("ButtonTest"));
     JButton viewButton = new JButton(Bundle.getMessage("ButtonView"));
     JTextField lengthField = new JTextField(10);
+    JTextField SensorDelay = new JTextField(5);
```

Comment:
This should be `sensorDelay` rather than `SensorDelay` (camelCased with leading lower case letter).

This is to avoid confusion between Classes and variables.